src/applications/disability-benefits/2346/sass/form-2346.scss

```diff
@@ -53,12 +53,12 @@ label {
 }
 
 .required {
-  color: $color-secondary-dark;
+  color: red;
```

Comment:
https://github.com/department-of-veterans-affairs/vets-website/pull/11999/files#r406826151